app_dart/lib/src/service/scheduler/files_changed_optimization.dart

```diff
@@ -74,18 +75,23 @@ final class FilesChangedOptimizer {
         log.warn('$refusePrefix: $reason');
         return FilesChangedOptimization.none;
       case SuccessfulFilesChanged(:final filesChanged):
-        if (filesChanged.length == 1 && filesChanged.contains('CHANGELOG.md')) {
-          return FilesChangedOptimization.skipPresubmitAllExceptFlutterAnalyze;
-        }
+        var markdownOnly = true;
         for (final file in filesChanged) {
           if (file == 'DEPS' || file.startsWith('engine/')) {
             log.info(
               '$refusePrefix: Engine sources changed.\n${filesChanged.join('\n')}',
             );
             return FilesChangedOptimization.none;
           }
+          if (markdownOnly && p.posix.extension(file) != '.md') {
```

Comment:
Today I learned: I am surprised it returns the '.' part of the extension. 

https://pub.dev/documentation/path/latest/path/extension.html